src/main/java/org/alliancegenome/curation_api/services/helpers/references/ReferenceSynchronisationHelper.java

```diff
@@ -88,26 +89,34 @@ protected Reference copyLiteratureReferenceFields(LiteratureReference litRef, Re
 		ref.setCurie(litRef.getCurie());
 		ref.setObsolete(false);
 
-		List<CrossReference> xrefs = new ArrayList<>();
+		Map<String, CrossReference> curationSystemXrefMap = new HashMap<>();
+		for (CrossReference xref : ref.getCrossReferences() ) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>')' is preceded with whitespace.